crates/rune/src/fmt/printer.rs

```diff
@@ -0,0 +1,2020 @@
+//! The `Printer` trait and implementations.
+
+use std::io::Write;
+
+use crate::ast::{
```

Comment:
I *usually* qualify types from here using `ast::AngleBracketed`. But hey, this works too 😆

Reply:
It became a bit too noisy :-)